packages/bpk-component-map/readme.md

```diff
@@ -0,0 +1,116 @@
+# bpk-component-map
+
+> Backpack map component.
+
+## Installation
+
+```sh
+npm install bpk-component-map --save-dev
+```
+
+## Usage
+
+```js
+import React from 'react';
+import BpkMap from 'bpk-component-map';
+
+export default () => (
+  <BpkMap centerLatitude={55.944357} centerLongitude={-3.1967116} />
+)
+```
+
+### with center latitude and longitude , zoom level Loading
+
+```js
+import React from 'react';
+import BpkMap from 'bpk-component-map';
+
+export default () => (
+  <BpkMap  
+    zoom={15}
+    centerLatitude={55.944357}
+    centerLongitude={-3.1967116}
+    zoomEnabled={false}
+    dragEnabled={false}
+    width="600px"
+    height="600px"
+    language="en" />
+)
+
+```
+
+### with bound box Loading
+
+```js
+import React from 'react';
+import BpkMap from 'bpk-component-map';
+
+export default () => (
+  <BpkMap  
+    boundSouth={55.94129273544452}
+    boundWest={-3.2285547854247625}
+    boundNorth={55.952707392208396}
+    boundEast={-3.159632742578083}
+    width="600px"
+    height="600px"
```

Comment:
What is the behaviour if the south-west-north-east boundaries don't form the same aspect ratio as width and height? will it be stretched, zoomed...  throw an error?

Reply:
it will be zoomed with suitable level.

Reply:
thanks.